website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md

```diff
@@ -119,56 +132,66 @@ To see specific tool parameters, select the task WDL link in the table; then fin
+For example, the Illumina sequencers used by the Broad Genomics Platform read 151 bp per read, of which 5 bp are reserved for the UMIs.
 
-The first step of the RNA with UMIs workflow is to use the fgbio’s [ExtractUMIsFromBam](http://fulcrumgenomics.github.io/fgbio/tools/latest/ExtractUmisFromBam.html) to remove UMI bases from the reads and store them in the RX read tag. 
+The first step of the RNA with UMIs workflow is to use the fgbio's [ExtractUMIsFromBam](http://fulcrumgenomics.github.io/fgbio/tools/latest/ExtractUmisFromBam.html) to remove UMI bases from the reads and store them in the RX read tag. 
 
-The resulting RX tag may contain information like “ACT-GCT.” The “ACT” is the 3 bp for read1 and the “GCT” is the 3 bp for read2. Even though we reserve 5 bases in a read, we only use 3 bases for UMI; the other two bases are reserved for cell barcode, which the workflow doesn’t use.
+The resulting RX tag may contain information like "ACT-GCT." The "ACT" is the 3 bp for read 1 and the "GCT" is the 3 bp for read 2. Even though we reserve 5 bases in a read, we only use 3 bases for UMI; the other two bases are reserved for cell barcode, which the workflow doesn't use.
 
 #### 3. Alignment with STAR
 
-After UMI extraction, the workflow aligns the paired-end reads to the reference (hg38 or hg19) using the [STAR aligner](https://github.com/alexdobin/STAR), which is specifically designed for RNA-seq data and is able to align cDNA sequences with many "gaps" that correspond to introns. 
+After UMI extraction, the workflow aligns the paired-end reads to the reference (hg38 or hg19) using the [STAR aligner](https://github.com/alexdobin/STAR), which is specifically designed for RNA-seq data and can align cDNA sequences with many "gaps" that correspond to introns. 
 
-After STAR alignment, the workflow outputs both a genome- and transcriptome-aligned BAM.
+The task uses the following parameters:
 
-<!--- STAR parameters will go here --->
+| Parameter | Value | Notes |
+| --- | --- | --- |
+| outSAMunmapped | Within | Includes unmapped reads in the output file rather than dropping those reads to facilitate potential downstream analysis. | 
+| outFilterMatchNminOverLread | 0.33 | Sets the fraction of reads that must match the reference. | <!--- this is likely being changed to default, waiting to hear from Takuto about this --->
+| outFilterMismatchNoverLmax | 0.1 | Sets the maximum allowable ratio of mismatches to read length. Reads with a ratio larger than the set value are filtered. For example, for paired-end reads with length 146, the reads are filtered if the number of mismatches is greater than 29 (146 \* 2 \* 0.1 = 29).|
+| alignEndsProtrude | 20 ConcordantPair | Allows a maximum of 20 protruding bases at alignment ends and marks these alignments as concordant pairs to prevent reads from small cDNA fragments that were sequenced into adapters from being dropped. This parameter allows for processing of FFPE and other low-quality or degraded data. |
```

Comment:
@takutosato Would you mind taking a look at this table to make sure this accurately captures the parameters we discussed as important for users?

Reply:
Did we define FFPE anywhere? Maybe we just say it "allows for the processing of data derived from low-quality or degraded tissue such as formalin-fixed paraffin-embedded samples.

Reply:
Good point - I'll update this.

Reply:
Updated.